### DISCUSSION TITLE
drill down on a repeated row panel

### DISCUSSION THREAD

**Discussion by p-malik** [grafana/grafana#38616] [Dashboards, Panels & Visualizations]
Hi, I wanted to check its possible to provide a click-through link functionality to the row panel itself, so that we can provide drill down based on the repeating row variable instead of doing that for each panel in the row?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
